pom.xml

```diff
@@ -23,17 +28,22 @@
         <dependency>
             <groupId>org.bukkit</groupId>
             <artifactId>bukkit</artifactId>
-            <version>1.7.2-R0.3</version>
+            <version>1.14.4-R0.1-SNAPSHOT</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.acrobot.chestshop</groupId>
             <artifactId>chestshop</artifactId>
-            <version>3.7.5</version>
+            <version>3.9.2</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/chestshop-3.9.2.jar</systemPath>
```

Comment:
This should use the ChestShop maven repository (and depend on the 1.13/1.14 version 3.10)